src/codec/cbor.js

```diff
@@ -40,7 +40,7 @@ export const encode = model => {
 
   return CBOR.encode({
     // leave out optionals unless they are set
-    ...(fct.length > 0 && { fct }),
+    ...(fct.length > 0 ? { fct }: { fct: [] }),
```

Comment:
`[]` should never be in the CBOR encoded format, it's part of the canonicalization and UCAN-IPLD spec.